frontend/front/src/redux/apiSlice.js

```diff
@@ -1,8 +1,18 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import Survey from "../dummyData/Survey.json";
+
+const mockSurvey = Survey;
+
+const mockBaseQuery =
```

Comment:
I spent maybe too much time trying to mock out the rtk-query query handler, but now we can use this to have the redux API stuff return whatever data we want for local testing purposes. I feel like getting the json-server up and running is maybe a more sustainable solution but this might help a bit until we're actually talking to the back-end live.